crates/cheatnet/src/rpc.rs

```diff
@@ -122,7 +141,7 @@ impl CallContractResult {
     fn from_execution_result(
         result: &EntryPointExecutionResult<CallInfo>,
         contract_address: &ContractAddress,
-    ) -> Self {
+    ) -> (Self, Vec<usize>) {
```

Comment:
Maybe we should add another function for getting this vector? `CallContractResult::from_...` doesn't seem like a place to return such tuple